ai_diffusion/model.py

```diff
@@ -622,6 +633,89 @@ def _finish_job(self, job: Job, event: ClientEvent):
             self.jobs.notify_cancelled(job)
             self.progress = 0
 
+    def _apply_resize_command(self, cmd: ResizeCommand, job: Job):
```

Comment:
This could just be
```py
job.params.resize_canvas = cmd.resize_canvas  # or cmd.extent if it stores width/height
```
Please add a `resize_canvas` field to job params, don't use metadata

Since then it's just 1 line probably don't need a function :)